src/lib.rs

```diff
@@ -70,6 +70,13 @@
 //! brew install sdl2
 //! ```
 //!
+//! Some systems (for example M1) require [setting
+//! `LIBRARY_PATH`](https://github.com/Rust-SDL2/rust-sdl2#homebrew) for the linker to find the
+//! installed library:
+//! ```bash
+//! export LIBRARY_PATH="$LIBRARY_PATH:$(brew --prefix)/lib"
+//! ```
+//!
```

Comment:
This reads better IMO:

```suggestion
//! Users of recent Homebrew versions will need to set `LIBRARY_PATH` for the linker to find the 
//! installed SDL2 package:
//!
//! ```bash
//! export LIBRARY_PATH="$LIBRARY_PATH:$(brew --prefix)/lib"
//! ```
//!
//! More information can be found in the 
//! [SDL2 documentation](https://github.com/Rust-SDL2/rust-sdl2#homebrew)
//!
```

If you change it, don't forget to regenerate the readmes.

Reply:
Yes, this more readable. Thank you! Sure thing. :)

I'd still like to discriminate between a default installation on macOS x86_64 and aarch64/ARM64. Because the former goes into `/usr/local`, therefor its libraries into `/usr/local/lib` which is already on the linker's search path by default and needs no further action. Are you ok with this nitpicking?

Reply:
> If you change it, don't forget to regenerate the readmes.

You wrote it, I read it but still missed it out in the first place. Hail to the CI setup which sorts out such issues! :)

Reply:
> Are you ok with this nitpicking?

Ah I didn't realise there was a difference. Yeah that's fine :). Perhaps:

```diff
- Users of recent Homebrew versions will need to set `LIBRARY_PATH` for the linker to find the 
+ Users of recent Homebrew versions on ARM Macs will need to set `LIBRARY_PATH` for the linker to find the 
```

There'll be an M2 or some other thing at some point, so I figured "ARM Macs" is better than "M1 Macs". Feel free to push a better change and I'll take a look at what you come up with :)

Reply:
> There'll be an M2 or some other thing at some point, so I figured "ARM Macs" is better than "M1 Macs". Feel free to push a better change and I'll take a look at what you come up with :)

I am with you and alredy it to _Apple silicon_ which [Apple uses itself](https://support.apple.com/en-us/HT211814) to refer to their own CPUs.